thali/thalicryptomanager.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var crypto = require('crypto');
+var fs = require('fs');
+var path = require('path');
+
+// The password is not secure because anyone who can get to the file can get
+// to the app and thus can get the password. The password is used here only to
+// satisfy the crypto/PKCS12 APIs.
+var password = 'password';
+var certname = 'certname';
+var country = 'country';
+var organization = 'organization';
+var pkcs12FileName = '/pkcs12.pfx';
+var macName = 'SHA256';
+var hashSizeInBytes = 16;
+
+module.exports = {
+  
+  pkcs12FileName: pkcs12FileName,
```

Comment:
We should never be exporting local variables. That just isn't right. At a minimum they should be wrapped in getter functions. But why are these here at all? Are you having binding problems? Is this for testing? Even for testing we should have setter wrappers with big warnings on them that they are only for testing purposes.